atom/browser/api/atom_api_window.cc

```diff
@@ -584,8 +584,17 @@ void Window::SetFocusable(bool focusable) {
   return window_->SetFocusable(focusable);
 }
 
-void Window::SetProgressBar(double progress) {
-  window_->SetProgressBar(progress);
+void Window::SetProgressBar(double progress, mate::Arguments* args) {
+  std::string mode;
+
+  if (args && args->Length() == 2) {
```

Comment:
Does it have to be followed strictly? Their code style is quite conservative, Electron could do some things differently than Chrome. I am also not a fan of the old school C-style `#ifdef` include guards, all decent compilers support `#pragma once`